docs/concepts/deployments.md

```diff
@@ -503,6 +507,30 @@ $ prefect deployment inspect 'Cat Facts/catfact'
 
 If you specify a schedule for a deployment, the deployment will execute its flow automatically on that schedule as long as a Prefect server and agent are running. Prefect Cloud creates schedules flow runs automatically, and they will run on schedule if an agent is configured to pick up flow runs for the deployment.
 
+### Create a flow run with an event trigger
+
+!!! cloud-ad "deployment triggers are only available in Prefect Cloud"
+
+Deployments can optionally take a trigger specification, which will configure an automation to run the deployment based on the presence or absence of events, and optionally pass event data into the deployment run as parameters via jinja templating.
+
+```yaml
+triggers:
+  - enabled: true
+    match:
+      prefect.resource.id: prefect.flow-run.*
+    expect:
+      - prefect.flow-run.Completed
+    match_related:
+      prefect.resource.id: prefect.flow.c5568579-9f63-415c-a53e-25f00fc35866
```

Comment:
I think we should make these examples use the `prefect.resource.name`, my thinking here is that oftentimes (at least in my usage) the flow might not even exist yet (though you'd likely know the name) or the flow might exist but to add this section you're going to have to go to the UI or CLI to lookup the id which both seem to semi-defeat the purpose of this shorthand.

Reply:
good call